documentation/modules/security/proc-accessing-kafka-using-ingress.adoc

```diff
@@ -3,43 +3,60 @@
 // assembly-configuring-kafka-listeners.adoc
 
 [id='proc-accessing-kafka-using-ingress-{context}']
-= Accessing Kafka using ingress
+= Accessing Kafka using ingresses
 
-This procedure shows how to access a Strimzi Kafka cluster from an external client outside of Kubernetes using Nginx Ingress.
+[role="_abstract"]
+Use an {NginxIngressController} to access a Strimzi Kafka cluster from clients outside the Kubernetes cluster. 
+The Nginx Ingress Controller supports connections from HTTP and HTTPS-based applications.
 
-To connect to a broker, you need a hostname (advertised address) for the Ingress _bootstrap address_,
-as well as the certificate used for authentication.
+To be able to use an Nginx Ingress Controller, add configuration for an `ingress` type listener in the `Kafka` custom resource. 
+When applied, the configuration creates a dedicated ingress and service for an external bootstrap and each broker in the cluster. 
+Clients connect to the bootstrap ingress, which routes them through the bootstrap service to connect to a broker. 
+Per-broker connections are then established using DNS names, which route traffic from the client to the broker through the broker-specific ingresses and services.
 
-For access using Ingress, the port is always 443.
+To connect to a broker, you specify a hostname (advertised address) for the ingress bootstrap address, as well as the certificate used for authentication.
+
+For access using an ingress, the port is always 443.
 
 .TLS passthrough
 
-Kafka uses a binary protocol over TCP, but the {NginxIngressController} is designed to work with the HTTP protocol.
-To be able to pass the Kafka connections through the Ingress, Strimzi uses the TLS passthrough feature of the {NginxIngressController}.
-Ensure TLS passthrough is enabled in your {NginxIngressController} deployment.
+Make sure that you enable TLS passthrough in your NGINX Ingress Controller deployment.
+Kafka uses a binary protocol over TCP, but the NGINX Ingress Controller is designed to work with a HTTP protocol. 
+To be able to route traffic through ingresses, Strimzi uses TLS passthrough with Server Name Indication (SNI).
 
-Because it is using the TLS passthrough functionality, TLS encryption cannot be disabled when exposing Kafka using `Ingress`.
+SNI helps with identifying and passing connection to Kafka brokers.
+In passthrough mode, TLS encryption is always used.
+Because connection passes to the brokers, the listeners use the TLS certificates signed by the internal cluster CA.
```

Comment:
```suggestion
Because the connection passes to the brokers, the listeners use the TLS certificates signed by the cluster CA and not the Ingress certificates.
```